db/import_test.go

```diff
@@ -893,6 +893,59 @@ func TestMetadataOnlyUpdate(t *testing.T) {
 
 }
 
+func TestImportResurrectionMou(t *testing.T) {
+	base.SetUpTestLogging(t, base.LevelInfo, base.KeyMigrate, base.KeyImport, base.KeyCRUD)
+	db, ctx := setupTestDBWithOptionsAndImport(t, nil, DatabaseContextOptions{})
+	defer db.Close(ctx)
+
+	collection := GetSingleDatabaseCollectionWithUser(t, db)
+
+	docID := "mouResurrection"
+
+	firstBody := Body{"foo": "bar"}
+	_, _, err := collection.Put(ctx, docID, firstBody)
+	require.NoError(t, err)
+
+	syncData, mou, _ := getSyncAndMou(t, collection, docID)
+	require.NotNil(t, syncData)
+	require.Nil(t, mou)
+
+	// Update via SDK, expect mou to be created
+	err = collection.dataStore.Set(docID, 0, nil, []byte(`{"foo": "baz"}`))
+	require.NoError(t, err)
+	base.RequireWaitForStat(t, func() int64 {
+		return db.DbStats.SharedBucketImport().ImportCount.Value()
+	}, 1)
+	syncData, mou, _ = getSyncAndMou(t, collection, docID)
+	if db.Bucket.IsSupported(sgbucket.BucketStoreFeatureMultiXattrSubdocOperations) {
```

Comment:
Should we just put a useMou() skip check at the start of this test?  It looks like the test is specifically checking _mou functionality.

Reply:
There is no code that tests this code pathway as is where we do an import -> ressurrection. It was not just that the mou wasn't deleted, it actually returned an `sgbucket.ErrDeleteXattrOnTombstone`. I'm inclined to leave it to ensure this pathway is successful in that case was well since it mirrors a QE test.